CRM/Admin/Form/SettingTrait.php

```diff
@@ -227,7 +227,7 @@ protected function addFieldsDefinedInSettingsMetadata() {
           $this->addRadio($setting, $props['title'], [1 => ts('Yes'), 0 => ts('No')], CRM_Utils_Array::value('html_attributes', $props), '&nbsp;&nbsp;');
         }
         elseif ($add === 'add') {
-          $this->add($props['html_type'], $setting, $props['title'], $options);
+          $this->add($props['html_type'], $setting, $props['title'], NULL, FALSE, $props['html_extra'] ?? NULL);
```

Comment:
Why are we no longer passing in $options - it seems like that would support other add types

Reply:
@eileenmcnaughton You are right, we can leave that in - PR now updated.